requirements.txt

```diff
@@ -1,3 +1,4 @@
-parcon ~= 0.1.25
+git+https://github.com/javawizard/parcon
```

Comment:
What was the issue here?

Reply:
The pypi version does not support Python3. javawizard/parcon#20.